### DISCUSSION TITLE
[FS2024] TBM 930 - Lights

### DISCUSSION THREAD

**Discussion by NeonRat85** [nguyenquyhy/Flight-Tracker-StreamDeck#431] [General]
Hey Folks!

I'm trying to configure some toggles for the Landing Lights and the Pulse Lights on the TBM 930 - I've tried the usual list in Variables and Events but no joy.

Is there a way we can log the activities in the sim e.g. using the console so I see what even fires when I flick the switches?

**Comment by nguyenquyhy** [comment]
You can use the log console feature of the free version of FSUIPC to see the events. If it needs Lvar then you can follow our MobiFlight guide to set that up.

**Reply by NeonRat85** [reply]
Thanks! I'll go grab that, log them and come back with the details!

**Comment by NeonRat85** [comment]
Okay - I've dug into FSUIPC and it doesn't register any of the buttons, e.g. NAV as an Event though I can make the Nav toggle with TOGGLE_NAV_LIGHTS so I might need to go down the lvar route, I'll go check the MobiFlight guidance

**Comment by NeonRat85** [comment]
Okay if anyone else is looking for the TBM 930 variables in 2024 this is what's been dug up so far in MobiFlight - not sure if this is the right place to post this?

---SOURCE GPU---
1 (>B:ELECTRICAL_CRASHBAR_Set)
2 (>B:ELECTRICAL_SOURCE_SWITCH_GPU)

1 (>B:ELECTRICAL_CRASHBAR_Set)
1 (>B:ELECTRICAL_SOURCE_SWITCH_BATT)

---SOURCE BATT---
0 (>B:ELECTRICAL_CRASHBAR_Set)
0 (>B:ELECTRICAL_SOURCE_SWITCH_OFF)

1 (>B:ELECTRICAL_CRASHBAR_Set)
1 (>B:ELECTRICAL_SOURCE_SWITCH_BATT)

---GEN---
0 (>B:ELECTRICAL_GENERATOR_SWITCH_Set) --- OFF
1 (>B:ELECTRICAL_GENERATOR_SWITCH_Set) --- MAIN
2 (>B:ELECTRICAL_GENERATOR_SWITCH_Set) --- STDBY

---STARTER---
0 (>B:ENGINE_STARTER_1_ON)
1 (>B:ENGINE_STARTER_1_OFF)
2 (>B:ENGINE_STARTER_1_ABORT)

---IGNIT---
0 (>B:ENGINE_IGNITION_1_Set) ---OFF
1 (>B:ENGINE_IGNITION_1_Set) ---ON
2 (>B:ENGINE_IGNITION_1_Set) ---ABORT

---TAXY LDG LIGHT---
0 (>B:TBM930_LIGHTING_TAXI_LANDING_LIGHT_SWITCH_Set)
1 (>B:TBM930_LIGHTING_TAXI_LANDING_LIGHT_SWITCH_Set)
2 (>B:TBM930_LIGHTING_TAXI_LANDING_LIGHT_SWITCH_Set) ---LDG

---NAV LIGHT---
0 (>B:TBM930_LIGHTING_NAVIGATION_LIGHT_SWITCH_Set)
1 (>B:TBM930_LIGHTING_NAVIGATION_LIGHT_SWITCH_Set)

---PULSE LIGHT---
0 (>B:TBM930_LIGHTING_PULSE_LIGHT_SWITCH_Set)
1 (>B:TBM930_LIGHTING_PULSE_LIGHT_SWITCH_Set)

---STOBE---
(>K:STROBES_OFF)
(>K:STROBES_ON)

---PRESSU BLEED---
2 (>B:TBM930_PRESSURIZATION_BLEED_AUTO)
0 (>B:TBM930_PRESSURIZATION_BLEED_OFF)

---BARO STD---
0 (>B:AUTOPILOT_BARO_1_SYNC_PUSH)
